app/src/main/res/values/strings.xml

```diff
@@ -24,4 +24,11 @@
     <string name="webview_out_of_date_title">WebView out-of-date</string>
     <string name="webview_out_of_date_message">Your current WebView version is %d. The WebView should be at least version %d for the PDF Viewer to work.</string>
     <string name="action_share">Share</string>
+
+    <string name="password_prompt_hint">Password</string>
+    <string name="password_prompt_description">Enter the password to decrypt this PDF file</string>
+    <string name="password_prompt_invalid_password">Invalid password</string>
+    <string name="open">open</string>
+    <string name="cancel">@android:string/cancel</string>
```

Comment:
It's better to not rely of framework's strings. If the device is set to an unsupported language, one button label will be in English and the other one will be translated.